src/test/java/com/queensburg/MainTest.java

```diff
@@ -10,4 +10,10 @@ public void someMethodAddsStuff() {
     int sum = Main.someMethod(3, 6);
     assertEquals("someMethod should sum the arguments", 9, sum);
   }
+
+  @Test
+  public void anotherMethodFindsTheDifference() {
```

Comment:
I don't like the name of this function. Maybe something like `totallyUselessMethodForThisProject`